CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ Most pull requests are merged with the "squash and merge" feature, so details ab
 
 It is unnecessary to manually edit (rebase) your commit history. If, however, you do want to save a pull request as multiple commits on `main`, ask me and we'll discuss. (The default branch is named `main`, not `master`.)
 
-### Building and teseting locally
+### Building and testing locally
```

Comment:
I added this one, the checks are not that smart. :)

Reply:
Thanks!